src/PanelGroup.js

```diff
@@ -379,13 +380,6 @@ var Panel = React.createClass({
 
 var Divider = React.createClass({
 
-  getDefaultProps: function() {
-    return {
-      dividerWidth: 1,
-      handleBleed: 4,
-    };
-  },
-
```

Comment:
That said, it looks like I messed up this PR by committing more changes to our fork's `master`. Let me know if you are okay with the original change, and I'll resubmit it as a new PR.